docs/Reference/Server.md

```diff
@@ -374,6 +375,28 @@ The possible values this property may have are:
   const fastify = require('fastify')({logger: customLogger});
   ```
 
+### `createRequestLogMessage`
+<a id="create-request-log-message"></a>
+
++ Default: `true`
+
+When logging is enabled via `disableRequestLogging` parameter, users can 
+implement the `createRequestLogMessage` function to customize their log management 
+system.  As a result, this setting allows users to produce a customizable log message 
+catered to their projects' needs, circumventing the default format. 
```

Comment:
```suggestion
When request logging is enabled, request logs can be customized by
supplying a `createRequestLogMessage(request, message)` function.
The function must be synchronous, and must not throw an error.
```

Reply:
Thank you @jsumners, I'll make sure incorporate this.